specification_documents/examples/intro_qc2.mzQC

```diff
@@ -55,26 +65,47 @@
           {
             "accession": "MS:4000060",
             "name": "number of MS2 spectra",
-            "value": 62299
+            "description": "The number of MS2 events in the run.",
+            "value": 62299,
+            "unit": {
```

Comment:
**issue:** These next 4 fours actually don't have a unit defined in the PSI-MS CV (they also predate the QC terms). Two related issues:
1. The validator still complains that a unit is missing, even though no unit is defined. This indicates a bug in the validator.
2. Should we propose to add units to pre-existing CV terms outside of the QC namespace?

Reply:
see https://github.com/MS-Quality-Hub/pymzqc/issues/47

Reply:
Units added in [psi-ms-CV#318](https://github.com/HUPO-PSI/psi-ms-CV/pull/318).